docs/zh_cn/understand_mmcv/data_transform.md

```diff
@@ -192,15 +192,15 @@ pipeline = [
                 dict(type='Rotate', angle=30)
             ],  # 第二种随机变换组合
         ],
-        pipeline_probs=[0.4, 0.6]  # 两种随机变换组合各自的选用概率
+        prob=[0.4, 0.6]  # 两种随机变换组合各自的选用概率
         )
     ...
 ]
 ```
 
 ### 多目标扩展（ApplyToMultiple）
 
-通常，一个数据变换类只会从一个固定的字段读取操作目标。虽然我们也可以使用 `Remap` 来改变读取的字段，但无法将变换一次性应用于多个字段的数据。为了实现这一功能，我们需要借助多目标扩展包装（`ApplyToMultiple`）。
+通常，一个数据变换类只会从一个固定的字段读取操作目标。虽然我们也可以使用 `ApplyToMapped` 来改变读取的字段，但无法将变换一次性应用于多个字段的数据。为了实现这一功能，我们需要借助多目标扩展包装（`ApplyToMultiple`）。
```

Comment:
It's really hard for me to grasp the meaning of `ApplyToMapped` without reading the documentation. I'd suggest `KeyMapper`, which is more clear and emphasizes its role as a helper/wrapper by the name itself.

Reply:
I think its simpler, let's try `KeyMapper`.